src/components/sound.js

```diff
@@ -83,27 +81,32 @@ module.exports.Component = registerComponent('sound', {
    */
   setupSound: function () {
     var el = this.el;
-    var listener;
     var sceneEl = el.sceneEl;
     var sound = this.sound;
 
+    // we want to make sure we have exactly one AudioListener. Let's stick it somewhere
+    // that's available to the next sound component that might get initialized
+    var listener = this.listener = sceneEl.audioListener || new THREE.AudioListener();
+    sceneEl.audioListener = listener;
+
     if (sound) {
       this.stop();
       el.removeObject3D('sound');
     }
 
-    listener = this.listener = new THREE.AudioListener();
-    sound = this.sound = new THREE.Audio(listener);
-    el.setObject3D('sound', sound);
+    function addListenerToCamera() {
+      sceneEl.camera.add(listener);
```

Comment:
The active camera can change. Do we have to change the listener when it happens?